src/sections/pages/orders/index.js

```diff
@@ -0,0 +1,190 @@
+import Status from './status';
+import UserCell from '../../../components/UserCell';
+import TablePagination from '../../../components/TablePagination';
+import './styles.css';
+
+const orders = [
+    {
+        user: { name: 'Sara', photoUrl: '/assets/images/user2.jpg' },
+        status: 'VERIFIED',
+        amount: '$144.55',
+        date: '11/07/2018',
+    },
+    {
+        user: { name: 'Reinier', photoUrl: '/assets/images/user1.jpg' },
+        status: 'PROCESSING',
+        amount: '$285.05',
+        date: '11/07/2018',
+    },
+    {
+        user: { name: 'Leo', photoUrl: '/assets/images/user3.jpg' },
+        status: 'PROCESSING',
+        amount: '$125.25',
+        date: '11/06/2018',
+    },
+    {
+        user: { name: 'Tahimi', photoUrl: '/assets/images/user4.jpg' },
+        status: 'EXPIRED',
+        amount: '$104.55',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Pepe' },
+        status: 'VERIFIED',
+        amount: '$124.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Jose' },
+        status: 'VERIFIED',
+        amount: '$144.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Pedro' },
+        status: 'PROCESSING',
+        amount: '$94.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Juan' },
+        status: 'PROCESSING',
+        amount: '$124.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Pablo' },
+        status: 'EXPIRED',
+        amount: '$104.55',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Maria' },
+        status: 'VERIFIED',
+        amount: '$124.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Juana' },
+        status: 'VERIFIED',
+        amount: '$144.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Julio' },
+        status: 'PROCESSING',
+        amount: '$94.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Olga' },
+        status: 'PROCESSING',
+        amount: '$124.05',
+        date: '11/04/2018',
+    },
+];
+
+export default class Orders extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { activePage: 1 };
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    getTableData() {
+        const { activePage } = this.state;
+        const firstItem = (activePage - 1) * 8;
+        const lastItem = firstItem + 8;
+        return orders.slice(firstItem, lastItem);
+    }
+
+    handleOnChange(event, page) {
+        this.setState({ activePage: page });
+    }
+
+    render() {
+        const { activePage } = this.state;
+        return (
+            <div className="react-rainbow-admin-orders_container">
+                <div className="react-rainbow-admin-orders_cards-container">
+                    <Breadcrumbs>
+                        <Breadcrumb label="Pages" href="/pages" />
```

Comment:
In the last comment here I said that: 
you need to use navigateTo function form history
remember that href refresh the page